modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin_{context}"]
+= Running your dynamic plug-in
+
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have an OpenShift cluster running.
+* You must have link:https://www.docker.com/[Docker] or link:https://podman.io/[Podman] installed.
+
+.Procedure
+
+In one terminal window:
+
+. Run the following commands:
++
+[source,terminal]
+----
+$ yarn install
+----
++
+[source,terminal]
+----
+$ yarn run start
+----
+
+In a second terminal window:
+
+[start=1]
+. Run the following commands:
++
+[source,terminal]
+----
+$ oc login
+----
++
+[source,terminal]
+----
+$ yarn run start-console
+----
+
+. Visit link:http://localhost:9000/example[local host] to view the running plug-in.
```

Comment:
shall we make `http://localhost:9000/example` plain link here? since some user may trying on a different machine and he can only knows the actual link by clicking `local host`, if we write `http://localhost:9000/example` here it would be quite straightforward

Reply:
Also I followed the exact same steps in this doc and I got following `404 Not Found` error when visiting `http://localhost:9000/example` , also I see some errors like
```
Error while loading plugin from /api/plugins/console-plugin-template/ a: GET request for "console-plugin-template" plugin failed: Get "http://host.containers.internal:9001/plugin-manifest.json": dial tcp 10.0.2.2:9001: connect: network is unreachable
```

Reply:
@yapei What podman version are you using? I suspect you might need to upgrade podman.

Reply:
You need at least 3.2.0: https://github.com/containers/podman/releases/tag/v3.2.0

> Podman now adds an entry to /etc/hosts, host.containers.internal, pointing to the current gateway (which, for root containers, is usually a bridge interface on the host system)

Reply:
@spadgett should I add 3.2.0 by Podman for clarity?

Reply:
@opayne1 I think so. I added it to the plugin template repo README as well.

Reply:
@spadgett  I'm running on Fedora and Docker is expected to be used, right? 
My docker version on fedora is:
```
$ docker version
Client:
 Version:           20.10.10
 API version:       1.41
 Go version:        go1.16.8
 Git commit:        485636f
 Built:             Sat Nov 13 14:50:36 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

```

My Podman version on fedora is:
```
$ podman version
Version:      3.4.2
API Version:  3.4.2
Go Version:   go1.16.8
Built:        Fri Nov 12 20:25:37 2021
OS/Arch:      linux/amd64
```